harness/src/utils.rs

```diff
@@ -128,6 +130,22 @@ pub fn kill_process_using_port(port: u16) {
         .expect("Failed to execute kill command");
 }
 
+pub async fn count_voter(raft: &Raft) -> usize {
+    let mut hash_set = HashSet::new();
+    let start_time = Instant::now();
+    while start_time.elapsed() < Duration::from_secs(1) {
+        raft.get_peers()
+            .await
+            .inner
+            .iter()
+            .filter(|(_, role)| role.role == InitialRole::Voter)
+            .for_each(|(id, _)| {
+                hash_set.insert(*id);
+            });
+    }
+    hash_set.len()
+}
```

Comment:
Is there a reason why the counting needs to be executed multiple times for 1 second, rather than just once after a 1-second delay?

Reply:
I use a loop because the tick interval is unpredictable, so repeatedly checking within the 2-second window ensures we capture all relevant state changes.

Reply:
`tick_interval` is fixed.
If you are referring to `election_tick`, we can fix it using `min_election_tick` and `max_election_tick`.

Reply:
Currently, tick_interval is fixed in this test. We need to modify it to handle varying tick_interval values